plugin/environment.go

```diff
@@ -213,6 +231,14 @@ func (env *Environment) GetManifest(pluginId string) (*model.Manifest, error) {
 }
 
 func (env *Environment) Activate(id string) (manifest *model.Manifest, activated bool, reterr error) {
+	defer func() {
+		if reterr != nil {
+			env.setPluginError(id, reterr.Error())
+		} else {
+			env.setPluginError(id, "")
```

Comment:
Are we doing this to clear an error string that might be there from before? (just checking)

Reply:
@cpoile Yes that's correct. `reterr == nil` means "Successful activation" so we should clear any existing error